plenum/common/messages/fields.py

```diff
@@ -368,16 +368,13 @@ def __init__(self, *args, **kwargs):
 
 
 class TimestampField(FieldBase):
-    _base_types = (float, int)
+    _base_types = (int,)
+    _oldest_time = 1499906902
```

Comment:
This is the time at which the network was bootstrapped, i will move that to a constant in config.